lib/cdo/poste.rb

```diff
@@ -23,7 +23,10 @@ def self.decrypt(encrypted)
   end
 
   def self.decrypt_id(encrypted)
-    return decrypt(encrypted).to_i
+    decrypt(encrypted).to_i
+  rescue OpenSSL::Cipher::CipherError, ArgumentError => e
+    CDO.log.warn "Unable to decrypt poste id: #{encrypted}. Error: #{e.message}"
```

Comment:
Not sure I have enough context to be certain. What triggers the exception? Could this be useful for debugging a hacking attempt?